docs/federation/types.md

```diff
@@ -85,6 +85,12 @@ An [ISO 8601][iso8601] date.
 
 Example: `2016-02-19`
 
+## Timezone
+
+A timezone in the form `Area/Location` as used in the [Time Zone Database][tz].
```

Comment:
This is based on a feature friendica already has: You can set something like "convert time to viewer timezone" (I don't remember the exact wording), so we decided to make this the default if no timezone is added and add the organizer timezone if it should be displayed always in this timezone.